### ISSUE TITLE
[Bug] UI state is not good after error in exporter

### ISSUE THREAD

**Issue by baruchiro** [brafdlog/caspion#248]
Steps to reproduce:

1. Set a not-existing directory for the CSV path.
2. Add YNAB exporter (you don't need to configure it)
3. RUN
4. Expect for error message about `no such file or directory`.
5. Click outside the error to back to the main view.

Now you may see the YNAB reporter is still loading, forever.

Also, the Importers are without the success/failed sign.

![image](https://user-images.githubusercontent.com/17686879/120004782-d9f11600-bfdf-11eb-882b-45a4bd08e9c2.png)

**Comment by Innush** [comment]
I can't reproduce.
I did the following steps:
1. I created a new folder, selected it in the CSV path input, and saved it.
2. I deleted the folder. 
3. Click on RUN.
Results: 
I got the "no such file or directory".
<img width="1434" alt="Screen Shot 2022-02-13 at 18 31 19" src="https://user-images.githubusercontent.com/33602729/153780372-6cab39bf-33b0-4c06-9e5b-bd02f950c981.png">

After clicking outside of the pop-up box and the YNAB account finished loading.
<img width="1447" alt="Screen Shot 2022-02-13 at 18 31 41" src="https://user-images.githubusercontent.com/33602729/153780437-fddbe2d7-06fb-4553-8118-7f1ca2130303.png">
